pkg/nuctl/command/export.go

```diff
@@ -94,8 +102,8 @@ func newExportFunctionCommandeer(exportCommandeer *exportCommandeer) *exportFunc
 		},
 	}
 
-	cmd.PersistentFlags().StringVarP(&commandeer.output, "output", "o", common.OutputFormatYAML, "Output format - \"yaml\", or \"json\"")
-	cmd.PersistentFlags().BoolVar(&commandeer.noScrub, "no-scrub", false, "Allow function sensitive data to be exported")
+	cmd.PersistentFlags().StringVarP(&commandeer.output, "output", "o", common.OutputFormatYAML, 'Output format - "json" or "yaml"')
```

Comment:
```suggestion
	cmd.PersistentFlags().StringVarP(&commandeer.output, "output", "o", common.OutputFormatYAML, "Output format - \"json\" or \"yaml\"")
```
This won't compile. Go doesn't support single quotation marks for strings. Please change back to double quotations with escaped quotations within the string.